compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/MockDesugar.java

```diff
@@ -317,6 +318,23 @@ private BLangFunctionBody generateBody() {
                 generateFieldBasedAccess("functionToMockPackage"),
                 generateRHSExpr(functionToMockPackageVal));
 
+        BLangTestablePackage bLangTestablePackage = this.bLangPackage.getTestablePkg();
+        List<BLangCompilationUnit> compUnitList = bLangTestablePackage.getCompilationUnits();
+
+        String mockFunctionClassList = "";
```

Comment:
What if we collect the class list to a List object instead of a string and construct the string before assigning it? WDYT?

Reply:
Yes. Initially, the idea I had was to include it in a list because it is a more straightforward and cleaner approach. 

However, I ran into problems creating the equivalent desugar code to include the various source files into the list. I also tried arrays but was running into compilation problems. 

This is why I resorted to passing a simple string and handling it at the `FunctionMock` class. Im looking into improving this.

Reply:
As discussed, I have passed an array instead of a string. This is cleaner than the previous approach